### PR TITLE
Update README for `v4` Release (Similar to Python README)

### PR DESCRIPTION
**Pull request by dvonthenen**
This updates the README in preparation for the `v4` release. This README is similar to the existing Python SDK README.

TODOs in another PR: Look into generating code docs from comments in code (similar to Go SDK). Issue: https://github.com/deepgram/deepgram-dotnet-sdk/issues/268

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,729 +7,232 @@ Official .NET SDK for [Deepgram](https://www.deepgram.com/). Power your apps wit
...
-    });
-```
-
-#### UrlSource
+# Getting an API Key
 
-| Property | Value  |          Description          |
-| -------- | :----- | :---------------------------: |
-| Url      | string | Url of the file to transcribe |
+🔑 To access the Deepgram API you will need a [free Deepgram API Key](https://console.deepgram.com/signup?jump=keys).
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 2]
```diff
@@ -7,729 +7,232 @@
...
-var deepgramClientOptions = new DeepgramClientOptions("apikey");
-var client = new PrerecordedClient(deepgramClientOptions,HttpClientFactory);
-var response = await client.TranscribeFileAsync(
-    stream,
-     new PrerecordedSchema()
-    {
-        Punctuate = true
-    });
-```
+This SDK supports the following versios:
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 3]
```diff
@@ -7,729 +7,232 @@
...
-| Keywords               | List<string> |          Keywords to which the model should pay particular attention to boosting or suppressing to help it understand context           |
-| InterimResults         | bool     |          Indicates whether the streaming endpoint should send you updates to its transcription as more audio becomes available          |
-| EndPointing            | string   |                             Indicates whether Deepgram will detect whether a speaker has finished speaking                              |
-| Channels               | int      |                               Number of independent audio channels contained in submitted streaming audio                               |
-| SampleRate             | int      |                Sample rate of submitted streaming audio. Required (and only read) when a value is provided for encoding                 |
-| Tag                    | List<string>   |                                                                                                                      |
+Or use the Nuget package Manager. Right click on project and select manage nuget packages.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 4]
```diff
@@ -7,729 +7,232 @@
...
 
+### Installing the Previous Version
 
-# Projects
+We guarantee that major interfaces will not break in a given major semver (ie `4.*` release). However, all bets are off moving from a `3.*` to `4.*` major release. This follows standard semver best-practices.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 5]
```diff
@@ -7,729 +7,232 @@
...
-
-Retrieves a specific project based on the provided projectId.
+# Quickstarts
 
-```csharp
-var result = await manageClient.GetProject(projectId);
-```
-
-[See our API reference for more info](https://developers.deepgram.com/reference/get-project).
+This SDK aims to reduce complexity and abtract/hide some internal Deepgram details that clients shouldn't need to know about.  However you can still tweak options and settings if you need.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 6]
```diff
@@ -7,729 +7,232 @@
...
-{
-    Start = Datetime.Now
-}
-var result = await manageClient.GetProjectUsageFieldsAsync(projectId,getProjectUsageFieldsSchema);
-```
+- PreRecorded Audio:
 
-#### UsageFieldsSchema
+    - Transcription From an Audio File - [examples/prerecorded/file](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/prerecorded/file/Program.cs)
+    - Transcrption From an URL - [examples/prerecorded/url](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/prerecorded/url/Program.cs)
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 7]
```diff
@@ -7,729 +7,232 @@
...
-var result = onPremClient.GetCredentialsASync(projectId,credentialsId);
-```
+# Logging
 
-## Remove Credentials
-remove credentials associated with the credentials id
-```csharp
-var result = onPremClient.DeleteCredentialsASync(projectId,credentialsId);
-```
+This SDK uses [Serilog](https://github.com/serilog/serilog) to perform all of its logging tasks. By default, this SDK will local `Information` level messages and higher (ie `Warning`, `Error`, etc) when you initialize thelibrary as follows:
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md]: will do!

File: README.md [hunk 8]
```diff
@@ -1,735 +1,238 @@
...
 
 ```bash
-dotnet test
-```
-
-Upon completion, a summary is printed:
-
-```bash
-Passed!  - Failed:     0, Passed:    69, Skipped:     0, Total:    69, Duration: 906 ms - Deepgram.Tests.dll (net7.0)
+pip install -r requirements-dev.txt
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:228]: will remove the paragraph

File: README.md [hunk 9]
```diff
@@ -1,735 +1,230 @@
...
 
-```csharp
-using Deepgram.CustomEventArgs;
-using Deepgram.Models;
-using System.Net.WebSockets;
+# Installation
 
-var deepgramClientOptions = new DeepgramClientOptions("apikey");
-var deepgramClient = new LiveClient(deepgramClientOptions);
+To install the latest version of the C# SDK using NuGet (latest means this version will guarantee change over time), run the following command from your terminal in your projects directory:
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:43]: will update

[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:43]: will update

File: README.md [hunk 10]
```diff
@@ -1,735 +1,230 @@
...
-| Tag                    | List<string>   |                                                                                                                      |
+Or use the NuGet package Manager. Right click on project and select manage NuGet packages.
 
+### Installing the Previous Version
 
-# Projects
+We guarantee that major interfaces will not break in a given major semver (ie, `4.*` release). However, all bets are off moving from a `3.*` to `4.*` major release. This follows standard semver best-practices.
 
-> projectId and memberId are of type `string`
+To install the previous major version of the .NET SDK, run the following command from your terminal in your projects directory:
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:55]: will update

File: README.md [hunk 11]
```diff
@@ -1,735 +1,230 @@
...
-
-Retrieves a specific project based on the provided projectId.
-
-```csharp
-var result = await manageClient.GetProject(projectId);
-```
+# Quickstarts
 
-[See our API reference for more info](https://developers.deepgram.com/reference/get-project).
+This SDK aims to reduce complexity and abstract/hide some internal Deepgram details that clients shouldn't need to know about.  However you can still tweak options and settings if you need.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:63]: will update

File: README.md [hunk 12]
```diff
@@ -1,735 +1,230 @@
...
 
-#### UsageFieldsSchema
+- PreRecorded Audio:
 
-| Property      | Value    |              Description               |
-| ------------- | :------- | :------------------------------------: |
-| Start         | DateTime | Start date of the requested date range |
-| End           | DateTime |  End date of the requested date range  |
+    - Transcription From an Audio File - [examples/prerecorded/file](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/prerecorded/file/Program.cs)
+    - Transcrption From a URL - [examples/prerecorded/url](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/prerecorded/url/Program.cs)
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:175]: will update

File: README.md [hunk 13]
```diff
@@ -1,735 +1,230 @@
...
-var result = onPremClient.GetCredentialsASync(projectId,credentialsId);
-```
+# Logging
 
-## Remove Credentials
-remove credentials associated with the credentials id
-```csharp
-var result = onPremClient.DeleteCredentialsASync(projectId,credentialsId);
-```
+This SDK uses [Serilog](https://github.com/serilog/serilog) to perform all of its logging tasks. By default, this SDK will local `Information` level messages and higher (ie `Warning`, `Error`, etc) when you initialize the library as follows:
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:202]: will update

File: README.md [hunk 14]
```diff
@@ -1,735 +1,230 @@
...
-
-```bash
-Passed!  - Failed:     0, Passed:    69, Skipped:     0, Total:    69, Duration: 906 ms - Deepgram.Tests.dll (net7.0)
-```
+[Code of Conduct](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/.github/CODE_OF_CONDUCT.md). Then see the
+[Contribution](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/.github/CONTRIBUTING.md) guidelines for more information.
 
 # Getting Help
 
 We love to hear from you so if you have questions, comments or find a bug in the
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:225]: will do

File: README.md [hunk 15]
```diff
@@ -1,735 +1,229 @@
 # Deepgram .NET SDK
 
-[![Nuget](https://img.shields.io/nuget/v/deepgram)](https://www.nuget.org/packages/Deepgram) [![Build Status](https://github.com/deepgram-devs/deepgram-dotnet-sdk/workflows/CI/badge.svg)](https://github.com/deepgram-devs/deepgram-dotnet-sdk/actions?query=CI) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](./.github/CODE_OF_CONDUCT.md) [![Discord](https://dcbadge.vercel.app/api/server/xWRaCDBtW4?style=flat)](https://discord.gg/xWRaCDBtW4)
+[![NuGet](https://img.shields.io/nuget/v/deepgram)](https://www.nuget.org/packages/Deepgram) [![Build Status](https://github.com/deepgram-devs/deepgram-dotnet-sdk/workflows/CI/badge.svg)](https://github.com/deepgram-devs/deepgram-dotnet-sdk/actions?query=CI) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](./.github/CODE_OF_CONDUCT.md) [![Discord](https://dcbadge.vercel.app/api/server/xWRaCDBtW4?style=flat)](https://discord.gg/xWRaCDBtW4)
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:3]: that is no longer true. removing that statement

File: README.md [hunk 16]
```diff
@@ -1,735 +1,229 @@
...
-var getProjectUsageFieldsSchema = new UsageFieldsSchema()
-{
-    Start = Datetime.Now
-}
-var result = await manageClient.GetProjectUsageFieldsAsync(projectId,getProjectUsageFieldsSchema);
-```
+    - Intent Recognition - [examples/analyze/intent](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/analyze/intent/Program.cs)
+    - Sentiment Analysis - [examples/sentiment/intent](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/analyze/sentiment/Program.cs)
+    - Summarization - [examples/analyze/intent](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/analyze/summary/Program.cs)
+    - Topic Detection - [examples/analyze/intent](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/analyze/topic/Program.cs)
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:170]: ooops. will fix.

File: README.md [hunk 17]
```diff
@@ -1,735 +1,229 @@
...
+    - Invitations - [examples/manage/invitations](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/invitations/Program.cs)
+    - Keys - [examples/manage/keys](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/keys/Program.cs)
+    - Members - [examples/manage/members](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/members/Program.cs)
+    - Projects - [examples/manage/projects](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/projects/Program.cs)
+    - Scopes - [examples/manage/scopes](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/scopes/Program.cs)
+    - Usage - [examples/manage/usage](https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/examples/manage/usage/Program.cs)
 
-# OnPrem
-OnPremClient methods
+To run each example set the `DEEPGRAM_API_KEY` as an environment variable, then `cd` into each example folder and execute the example: `dotnet run <Project File>.csproj`.
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply README.md:197]: will update

**Review by jpvajda — APPROVED** [review]

**Pull request merged**